src/shims/unix/linux/foreign_items.rs

```diff
@@ -72,6 +72,60 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
             }
 
             // Threading
+            "prctl" => {
```

Comment:
Note that this here will *not* be used on Android -- Linux is just the "normal" Linux stack, from which Android has diverged too far.

Not sure if it's worth supporting this on regular Linux targets. For now, I'd say just move this to `android/foreign_items.rs`. We can always move it to some shared place later.

Reply:
By the way, it can be also used on FreeBSD which isn't Linux, but not on macOS which FreeBSD based.